src/explorer/AzureAccountTreeItem.ts

```diff
@@ -54,4 +40,47 @@ export class AzureAccountTreeItem extends AzureAccountTreeItemBase {
         }
         return super.compareChildrenImpl(item1, item2);
     }
+
+    public async pickTreeItemImpl(expectedContextValues: (string | RegExp)[]): Promise<AzExtTreeItem | undefined> {
+        const subscription: string = localize('subscription', 'subscription');
+        const subscriptionOrTrialApp: string = localize('subscriptionOrTrialApp', 'subscription or trial app');
+        const trialAppOrSignIn: string = localize('trialAppOrSignIn', 'trial app, sign in, or create a free Azure account');
+
+        if (expectedContextValues.includes(TrialAppTreeItem.contextValue) && this.trialAppNode) {
+            if (this.isLoggedIn) {
```

Comment:
nit: I prefer

`this.childTypeLabel = this.isLoggedIn ? subscriptionOrTrialApp : trialAppOrSignIn`